LoopKitUI/Views/ScheduleEditor.swift

```diff
@@ -147,26 +147,31 @@ struct ScheduleEditor<Value: Equatable, ValueContent: View, ValuePicker: View, A
                 .navigationBarBackButtonHidden(true)
                 .navigationBarItems(leading: backButton, trailing: trailingNavigationItems)
         case .settings:
-            pageWithCancel
+            page
```

Comment:
[nit] This also...how does the code in this file avoid the bouncing?

Reply:
The views here are relying on structural identity for swiftui to know when to update vs when to recreate. In the previous code here, the pageWithCancel was returning views of different types when the cancel button was shown vs when it wasn't. The change here fixes the improper use of AnyView, and also creates a stable structural identity for page. Before, the entire page was losing its identity and being recreated.